lib/python/CMakeLists.txt

```diff
@@ -53,10 +53,6 @@ target_compile_definitions(${PYTHON_TARGET_NAME}
 # -----------------------------------------------------------------------------
 # include directories
 # -----------------------------------------------------------------------------
```

Comment:
Should this comment block get deleted too?

Reply:
Ah, Thanks @mattyjams . Good point, please see 14218defb8